src/framework/components/element/text-element.js

```diff
@@ -1120,6 +1107,53 @@ Object.assign(pc, function () {
         _shouldAutoFit: function () {
             return this._autoFitWidth && !this._autoWidth ||
                    this._autoFitHeight && !this._autoHeight;
+        },
+
+        // calculate the number of characters per texture up to, but not including
+        // the specified symbolIndex
+        _calculateCharsPerTexture: function (symbolIndex) {
```

Comment:
symbolIndex arg doesn't seem to be used can we remove it?

Reply:
Not sure I understand... symbolIndex index is used in the function and two of the callsites...